docs/UserGuide.md

```diff
@@ -38,12 +60,29 @@ optimized for use via a Command Line Interface (CLI).
 5. Run the WerkIt! application with the command: `java -jar WerkIt.jar`.
 
 ---
-
 ## Features 
+In this section, features of the application can be classified into 4 main categories. They are: <br>
+* [Exercise and Workout Features](#exercise-and-workout-features)
+* [Plan Features](#plan-features) 
+* [Schedule Features](#schedule-features)
+* [Search Features](#search-features)
 
-> **Heads up!** Your inputs cannot contain the pipe character `|`!
+Each feature is being run by a specific command format. When the command format is matched, 
+the command will then be processed by the application and produce the intended outcome. The format of the commands
+contains two types of formatting.
+
+| Format         | Parameters                                                                                                                               |
+|----------------|------------------------------------------------------------------------------------------------------------------------------------------|
+| `<condition>`  | Contents enclosed between "<>" are the parameters needed for the command to be valid. <br /> Not all commands have conditions to be met. |
+| `/commandType` | Content after "/" is to classify the command type. <br/> Such as classifying it to be a workout/ plan/ schedule/ search command          |
```

Comment:
Perhaps the extra spaces after the forward slashes can be omitted?

So it will look like "...it to be a **workout/plan/schedule/search** command".

Reply:
i tried omitting it but the overview of the sentence will look very cramp